models/articles.models.js

```diff
@@ -51,15 +32,58 @@ exports.fetchArticleByIdToPatch = (id, incrementVotes) => {
       return rows[0];
     });
 };
-exports.fetchAllArticles = () => {
-  let queryStr = `SELECT articles.*, COUNT(comments.comment_id) AS comment_count
-  FROM articles
-  LEFT JOIN comments ON articles.article_id = comments.article_id  
-  GROUP BY articles.article_id
-  ORDER BY created_at DESC`;
-  return db.query(queryStr).then((body) => {
-    return body.rows;
-  });
+exports.fetchAllArticles = (sort_by = "created_at", order = "DESC", topic) => {
+  return db
+    .query(`SELECT topic FROM articles GROUP BY topic`)
```

Comment:
We really want to SELECT slug FROM topics instead of selecting topic from articles here, as if a new topic is added to your db it will appear in the topics table first (and would still be a valid option even if it had no relevant articles yet)